t/cmd/lfstest-gitserver.go

```diff
@@ -749,6 +749,24 @@ func storageHandler(w http.ResponseWriter, r *http.Request) {
 					byteLimit = 8
 					batchResumeFailFallbackStorageAttempts++
 				}
+			} else if len(by) == len("status-batch-retry") && string(by) == "status-batch-retry" {
+				if rangeHdr := r.Header.Get("Range"); rangeHdr != "" {
+					regex := regexp.MustCompile(`bytes=(\d+)\-(.*)`)
+					match := regex.FindStringSubmatch(rangeHdr)
+					if match != nil && len(match) > 2 && len(match[2]) != 0 {
+						first, _ := strconv.ParseInt(match[1], 10, 32)
+						second, _ := strconv.ParseInt(match[2], 10, 32)
+						if second < first {
```

Comment:
Do you think it would be worth adding a comment indicating what it means to be in this conditional?

Reply:
Sure, I can add comments for these conditionals.